frontend/src/pages/Overview/OverviewPage.tsx

```diff
@@ -957,7 +957,9 @@ export class OverviewPage extends React.Component<OverviewProps, State> {
                                 title={ns.name}
                               >
                                 {ns.name}
-                                {ns.name === serverConfig.istioNamespace && <ControlPlaneBadge></ControlPlaneBadge>}
+                                {ns.name === serverConfig.istioNamespace && (
```

Comment:
Now in multi cluster, we have the possibility to have a remote cluster with the istioNamespace but no Istiod, what do you think to extend this check to this condition? Or maybe we can think about this when the Overview page is re worked?

Reply:
I would leave that for next iteration with Overview rework. To keep this PR only for adding cluster param.